ui/src/main/resources/edu/wpi/grip/ui/templates/java/operations/Find_Blobs.vm

```diff
@@ -1,14 +1,51 @@
-	public static class Blob {
-		public final double x, y, size;
-		
-		protected Blob(double x, double y, double size) {
-			this.x = x;
-			this.y = y;
-			this.size = size;
-		}
-	}
-
- 	public static void ${step.javaName()}(Mat input,Integer minArea,double[] Circularity,Boolean darkBlobs,
- 		List<Blob> blobList){
-
-	}
\ No newline at end of file
+    public static class Blob {
+        public final double x, y, size;
+
+        protected Blob(double x, double y, double size) {
+            this.x = x;
+            this.y = y;
+            this.size = size;
+        }
+    }
+    public static void ${step.javaName()}(Mat input, double minArea, double[] circularity,
+    Boolean darkBlobs, MatOfKeyPoint blobList){
+        FeatureDetector blobDet = FeatureDetector.create(FeatureDetector.SIMPLEBLOB);
+
+        try {
+            File tempFile = File.createTempFile("config", ".xml");
+
+            StringBuilder config = new StringBuilder();
+
+            config.append("<?xml version=\"1.0\"?>\n");
+            config.append("<opencv_storage>\n");
+            config.append("<thresholdStep>10.</thresholdStep>\n");
+            config.append("<minThreshold>50.</minThreshold>\n");
+            config.append("<maxThreshold>220.</maxThreshold>\n");
+            config.append("<minRepeatability>2</minRepeatability>\n");
+            config.append("<minDistBetweenBlobs>10.</minDistBetweenBlobs>\n");
+            config.append("<filterByColor>1</filterByColor>\n");
+            config.append("<blobColor>"+(darkBlobs ? 0 : 255)+"</blobColor>\n");
```

Comment:
Don't use String concatenation when you're already using a `StringBuilder`, it will create a new `StringBuilder` for each call.

So 
`"<blobColor>"+(darkBlobs ? 0 : 255)+"</blobColor>\n"`
becomes
`new StringBuilder("<blobColor>").append(darkBlobs ? 0 : 255).append("</blobColor>\n").toString()`

You'll end up creating a bunch of redundant `StringBuilder` instances